tests/python/relay/test_pass_alter_op_layout.py

```diff
@@ -31,6 +31,17 @@ def run_opt_pass(expr, passes):
     entry = mod["main"]
     return entry if isinstance(expr, relay.Function) else entry.body
 
+def reset_retrieve_alter_op_layout(op_name, alter_layout):
```

Comment:
Consider use RAII Pattern, this will allow the code to exit correctly even if the testcase raises an exception in the middle of the body

```
with TempOpAttr(name, attr):
     body
```

Reply:
Now I understood what you meant. Thanks for your patience (and teaching new practices) :)